testproto/testproto.go

```diff
@@ -0,0 +1,101 @@
+package testproto
```

Comment:
This testing package really needs comments, as well as a README-style package level comment explaining what it is, why it exists, and giving a brief rundown on how to use it.